pkg/apis/admission/types.go

```diff
@@ -20,6 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/consts/admission"
```

Comment:
this is a questionable top-level component. I wouldn't really want to add a new repo and dependency just for this.